app/Http/Transformers/ChannelEpisodesTransformer.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Suyabay\Http\Transformers;
+
+use League\Fractal;
+use Suyabay\Episode;
+
+class ChannelEpisodesTransformer extends Fractal\TransformerAbstract
```

Comment:
**Code Quality - Nitpick**
@andela-tolotin You are not Implementing Fractals the right way. Lets see on this.